contracts/contracts/extensions/RoleManagementFacet.sol

```diff
@@ -3,9 +3,10 @@ pragma solidity 0.8.18;
 
 import {IRoles} from './Interfaces/IRoles.sol';
 import {IRoleManagement} from './Interfaces/IRoleManagement.sol';
-import {SupplierAdmin} from './SupplierAdmin.sol';
+import {SupplierAdminFacet} from './SupplierAdminFacet.sol';
+import {_ROLE_MANAGEMENT_RESOLVER_KEY} from '../constants/resolverKeys.sol';
 
-abstract contract RoleManagement is IRoleManagement, SupplierAdmin {
+contract RoleManagementFacet is IRoleManagement, SupplierAdminFacet {
```

Comment:
This is incorrect. A facet never extends to other Facets:
`contract RoleManagementFacet is IRoleManagement, SupplierAdminStorageWrapper {`